build/generate-opts.py

```diff
@@ -371,7 +372,19 @@ def __init__(self, items, **defaults):
         hint_option,
         let_option,
         comment_option_since_4_4,
-    ], opts_name='extra'))
+    ], opts_name='extra')),
+
+    ('mongoc_count_document_opts_t', Struct([
+        read_concern_document_option,
+        session_option,
+        collation_option,
+        server_option,
+        ('skip', {'type': 'bson_value_t', 'help': 'An int specifying how many documents matching the ``query`` should be skipped before counting.'}),
+        ('limit', {'type': 'bson_value_t', 'help': 'An int specifying the maximum number of documents to count.'}),
+        comment_option_string_pre_4_4,
+        hint_option
```

Comment:
I noticed `maxTimeMS` isn't listed here, although it's also absent from other structs. Is that because it's accepted via `accept_extra=True` (default) and just undocumented in the man pages?

Reply:
I expect `maxTimeMS` is included in `aggregate_options` passed to `mongoc_collection_read_command_with_opts` [here](https://github.com/kevinAlbs/mongo-c-driver/blob/18117ba989953e46398c5d0ef9fe445263a46cbb/src/libmongoc/src/mongoc/mongoc-collection.c#L961-L962).

https://github.com/mongodb/mongo-c-driver/pull/1267/commits/3caa0154c2a8888a3bf32127439a049002b3889e adds a test to verify `maxTimeMS` is included in the `aggregate` command.